client/app/pages/dashboards/dashboard.js

```diff
@@ -133,9 +133,19 @@ function DashboardCtrl(
     this.globalParameters = this.dashboard.getParametersDefs();
   };
 
-  $scope.$on('dashboard.update-parameters', () => {
-    this.extractGlobalParameters();
-  });
+  // ANGULAR_REMOVE_ME This forces Widgets re-rendering
+  // use state when Dashboard is migrated to React
+  this.forceDashboardGridReload = () => {
+    this.dashboard.widgets = [...this.dashboard.widgets];
```

Comment:
Eventually had to change to use this to force the components re-rendering. @kravets-levko in case you have a better idea :)

Reply:
I think it's totally fine :+1: